packages/webdriver/src/index.js

```diff
@@ -14,7 +14,12 @@ import AppiumProtocol from '../protocol/appium.json'
 export default class WebDriver {
     static async newSession (options = {}, modifier, proto = {}, commandWrapper) {
         const params = validateConfig(DEFAULTS, options)
-        logger.setLevel('webdriver', params.logLevel)
+
+        if (logger.setLevel) {
```

Comment:
Let's move this to the `@wdio/logger` package so that whoever consumes this expects a `setLevel` to be available if though the web version of it is a noop

Reply:
👍